pages/posts/2024-portal-prague.md

```diff
@@ -0,0 +1,195 @@
+based approach to archival storage.  This model would reduce the overall
+storage requirements for the network and be a building block for more efficient
+syncing of archival nodes.  This topic is still being actively researched.
+
+
+## Verkle & Portal
+
+![Milos at the whiteboard](../../public/images/2024-prague-milos-whiteboard.jpg)
+
+Related to state data, we also spend some time discussing Verkle tries which
+will change to how Ethereum stores state data. Milos Stankovic has been doing
+research on how Portal can handle Verkle.  Luckily, most of our plans for how
+to handle state in the current Merkle context apply directly to how it looks
+like state will be handled under Verkle. There are still a few unsolved
+problems surrounding how to efficiently implement archival storage under
+Verkle.
+
+
+## 4444s and Execution Clients
+
+![EL Adoption](../../public/images/2024-prague-el-adoption.jpeg)
+
+[EIP-4444](https://eips.ethereum.org/EIPS/eip-4444) was originally authored in
+2021 and advocates for execution layer (EL) clients to drop chain history for
+block that are sufficiently old.  Since then, the conversation has been largely
+centered around the alternative data sources that are needed in order to ensure
+that once clients drop this data that we can still reliably retrieve it.
+Portal network has been aiming at serving as part of the 4444s solution, with
+our history network being well suited for storing and serving individual block
+data.
+
+In the recent ACD interop event in Kenya the EL teams got together to discuss
+finalizing 4444s plans and implementing the functionality in their individual
+clients. During our event this week we discussed loose plans that we'll be
+proposing to EL teams to provide them with support in integrating with the
+Portal history network with the expressed goal of delivering 4444s this year.
+
+## Glados
+
+![Glados plans](../../public/images/2024-prague-glados-plans.jpeg)
+
+Glados is our network monitoring infrastructure. It provide visibility into the
+current and historical status of our live networks. During the summit we
+discussed future plans for more graphs and information we want from Glados and
+new types of monitoring that we want to be possible.  Glados has continued to
+be a key driver in our development process. It tells us important metrics about
+our network as well as helping us identify problems as they occur.
+
+
+## Longer Term Designs and Plans
+
+During our discussions we covered a number of research topics focused on
+solving longer term problems and protocol designs.  Some of the "simpler"
+things in this category are ideas around how to support "super" nodes on our
+network that store an above average amount of data.  The way our DHT storage
+and retrieval mechanism is built makes it discover content on nodes that are
+very far away from that content address.  This is simply an inherent property
```

Comment:
Not sure I understand this sentence, don't you mean very close to the content address?